docs/codeql/codeql-language-guides/codeql-library-for-actions.rst

```diff
@@ -0,0 +1,136 @@
+.. _codeql-library-for-actions:
+
+CodeQL library for GitHub Actions
+=======================
+
+When you're analyzing GitHub Actions workflows and action metadata files, you can make use of the large collection of classes in the CodeQL library for GitHub Actions.
```

Comment:
```suggestion
When you're analyzing GitHub Actions workflows and Action metadata files, you can make use of the large collection of classes in the CodeQL library for GitHub Actions.
```

we are capitalizing below, so suggesting here too.

Reply:
Makes sense. I was trying to distinguish between Actions (product) and Action (component).